src/components/OffersNumber/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { Flex, Text, Title } from "@mantine/core";
+
+export default function OffersNumber({
+  offer,
+}: {
+  offer: { number: string; title: string };
+}) {
```

Comment:
Same thing here, just call props as offer then you can do stuff like `<OffersNumber number={69} title="God" />`

Reply:
yea i was thinking of doing like that, but then for some reasons i dropped that idea